airbyte-workers/src/main/java/io/airbyte/workers/process/KubeProcessBuilderFactoryPOC.java

```diff
@@ -47,15 +41,16 @@
 
   public static void main(String[] args) throws InterruptedException, IOException {
     LOGGER.info("Launching source process...");
-    Process src = new KubePodProcess(KUBE_CLIENT, "src", "default", "np_source:dev", 9002, false);
+    Process src = new KubePodProcess(KUBE_CLIENT, "src", "default", "np_source:dev", 9002, 9003, false);
 
     LOGGER.info("Launching destination process...");
-    Process dest = new KubePodProcess(KUBE_CLIENT, "dest", "default", "np_dest:dev", 9003, true);
+    // Process dest = new KubePodProcess(KUBE_CLIENT, "dest", "default", "worker-test:print-to-stdout",
+    // 9004, 9005, true);
 
     LOGGER.info("Launching background thread to read destination lines...");
     ExecutorService executor = Executors.newSingleThreadExecutor();
     var listenTask = executor.submit(() -> {
-      BufferedReader reader = new BufferedReader(new InputStreamReader(dest.getInputStream()));
+      BufferedReader reader = new BufferedReader(new InputStreamReader(src.getErrorStream()));
```

Comment:
In combination with the changes in the dockerfile, I'm seeing 
```
2021-05-26 14:36:17 INFO i.a.w.p.KubeProcessBuilderFactoryPOC(lambda$main$0):58 - {} - Destination sent: {"msg": "one"}
2021-05-26 14:36:17 INFO i.a.w.p.KubeProcessBuilderFactoryPOC(lambda$main$0):58 - {} - Destination sent: {"msg": "two"}
2021-05-26 14:36:18 INFO i.a.w.p.KubeProcessBuilderFactoryPOC(lambda$main$0):58 - {} - Destination sent: {"msg": "three"}
2021-05-26 14:36:19 INFO i.a.w.p.KubeProcessBuilderFactoryPOC(lambda$main$0):58 - {} - Destination sent: {"msg": "four"}
2021-05-26 14:36:20 INFO i.a.w.p.KubeProcessBuilderFactoryPOC(lambda$main$0):58 - {} - Destination sent: {"msg": "five"}
2021-05-26 14:36:21 INFO i.a.w.p.KubeProcessBuilderFactoryPOC(lambda$main$0):58 - {} - Destination sent: {"msg": "six"}
2021-05-26 14:36:22 INFO i.a.w.p.KubeProcessBuilderFactoryPOC(lambda$main$0):58 - {} - Destination sent: {"msg": "seven"}
2021-05-26 14:36:23 INFO i.a.w.p.KubeProcessBuilderFactoryPOC(lambda$main$0):58 - {} - Destination sent: {"msg": "eight"}
```
in logs, which confirms this is working.